### ISSUE TITLE
Publish on Stackage

### ISSUE THREAD

**Issue by bb010g** [SFML-haskell/SFML#20]

**Comment by sulami** [comment]
I guess I could handle this as well, but I am not sure if Stackage is okay with external build dependencies?

**Comment by jeannekamikaze** [comment]
I would have to look into Stack first, as I am not familiar with it. What dependencies are you worried about, the C libs?

**Comment by sulami** [comment]
Yes, exactly. I had a look at various packages today and there are plenty of bindings that depend on external libraries to build, so I guess it would be no problem. I just thought it might go against the stability mantra.

**Comment by RyanGlScott** [comment]
Generally, it's not an issue. The Stackage nightly builds test everything in an Ubuntu 16.04 environment, so as long as you can `apt-get install` your dependencies, it's possible to add to Stackage. And in the case of SFML,

```
apt-get install libcsfml-dev libsfml-dev
```

suffices.

Just open a PR against https://github.com/fpco/stackage, add your package in https://github.com/fpco/stackage/blob/master/build-constraints.yaml, and add `libcsfml-dev` and `libsfml-dev` to https://github.com/fpco/stackage/blob/master/debian-bootstrap.sh.
